### PR TITLE
Add system.close_orphaned_rooms event

### PR DESCRIPTION
**Pull request by Leonqn**

### CODE REVIEW

File: src/event.rs [hunk 1]
```diff
@@ -69,7 +69,7 @@ impl Event {
                 payload,
                 build_props("system.close_orphaned_rooms"),
                 &svc_account(config, "conference"),
-                CONFERENCE_API_VERSION,
+                "v1",
```
Comment (khodzha) [inline src/event.rs:72]: why would we do this? :thinking:

**Review by khodzha — APPROVED** [review]

File: src/event.rs (hunk 1 above)
Comment (Leonqn) [reply src/event.rs:72]: Do you mean v1? Conference still has v1 api version and I think it can't accept messages sent to v2?

Comment (khodzha) [reply src/event.rs:72]: i meant moving from constant to literal

Comment (Leonqn) [reply src/event.rs:72]: It is not moving, in constant we have "v2"

Comment (Leonqn) [reply src/event.rs]: Just renamed constant and made another one for v1

**Review by manifest — APPROVED** [review]

**Pull request merged**